### PR TITLE
Added logout page

### LINKED ISSUE #2: Login and Register pages

### PR DESCRIPTION
**Pull request by japhethbalane**
Screenshot from 2023-10-25 22-19-43

### CODE REVIEW

**Review by PCalape — APPROVED** [review]

File: src/routes/logout/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		<p class="mt-2 text-slate-800 mb-8">Transactions Tracking. Simplified.</p>
+		<Alert type="success">
+			<p>
+				You have successfully logged out from your account. <a href="/" class="underline">Click here</a> to login to your account
+				again.
+			</p>
+		</Alert>
+	</div>
+	<p class="absolute bottom-7 text-xs text-gray-400">
+		Developed by <a href="/" class="underline">Black Flower</a>
```
Comment (arnellebalane) [inline src/routes/logout/+page.svelte:19]: @japhethbalane Link this to our Github organization page.

File: src/routes/logout/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		<h1 class="text-5xl font-extrabold text-slate-800">pitaka</h1>
+		<p class="mt-2 text-slate-800 mb-8">Transactions Tracking. Simplified.</p>
+		<Alert type="success">
+			<p>
+				You have successfully logged out from your account. <a href="/" class="underline">Click here</a> to login to your account
+				again.
+			</p>
+		</Alert>
+	</div>
+	<p class="absolute bottom-7 text-xs text-gray-400">
```
Comment (arnellebalane) [inline src/routes/logout/+page.svelte:18]: @japhethbalane So we don't keep repeat this footer in all pages, should we put this in the layout that wraps all the pages? But you can do it in a followup PR.

**Review by arnellebalane — APPROVED** [review]

File: src/routes/logout/+page.svelte (hunk 1 above)
Comment (japhethbalane) [reply src/routes/logout/+page.svelte:19]: Added a task for this, and will update this on a separate PR :+1:

File: src/routes/logout/+page.svelte (hunk 2 above)
Comment (japhethbalane) [reply src/routes/logout/+page.svelte:18]: Added a task for this, and will update this on a separate PR :+1:

**Pull request merged**